pkgs/applications/audio/cdparanoia/default.nix

```diff
@@ -62,13 +63,77 @@ stdenv.mkDerivation rec {
         hash = "sha256-krfprwls0L3hsNfoj2j69J5k1RTKEQtzE0fLYG9EJKo=";
       })
     ]
-    ++ lib.optional stdenv.hostPlatform.isMusl ./utils.patch;
+    ++ lib.optional stdenv.hostPlatform.isMusl ./utils.patch
+    ++ lib.optionals stdenv.hostPlatform.isFreeBSD [
```

Comment:
We cannot make those patches unconditional? Otherwise they could easily break with a future update.

Reply:
The patches needed some massaging, but I was able to make them unconditional.